packages/api/test/e2e/promise-tx.spec.ts

```diff
@@ -93,27 +93,33 @@ describe.skip('e2e transactions', () => {
       .signAndSend(keyring.dave.address(), logEvents(done));
   });
 
-  it('makes a transfer (signAndSend via Signer) - sad path', async () => {
-    //no signer
-    api.setSigner();
+  it('makes a transfer (signAndSend via Signer) with undefined Signer', async () => {
+    const signer: any = undefined;
+    // no signer
+    api.setSigner(signer);
 
     await expect(api.tx.balances
       .transfer(keyring.eve.address(), 12345)
       .signAndSend(keyring.alice.address())).rejects.toThrow('no signer exists');
+  });
 
+  it('makes a transfer (signAndSend via Signer) with the wrong keyring pair', async () => {
     const signer = new SingleAccountSigner(keyring.dave);
 
     api.setSigner(signer);
 
-    //no callback
+    // no callback
     await expect(api.tx.balances
       .transfer(keyring.eve.address(), 12345)
       .signAndSend(keyring.alice.address())).rejects.toThrow('does not have the keyringPair');
+  });
 
-    //with callback
+  it('makes a transfer (signAndSend via Signer)  with the wrong keyring pair with a callback', async () => {
+    // with callback
     await expect(api.tx.balances
       .transfer(keyring.eve.address(), 12345)
-      .signAndSend(keyring.alice.address(), ({ events, status }) => {
+      .signAndSend(keyring.alice.address(), (cb: any) => {
+        expect(cb).toBeInstanceOf(Object);
```

Comment:
I think we should not expect anything here: if there's something, the test should fail

Reply:
The test is explicitly `makes a transfer (...) with a callback`.
Tslint complains when the block is empty.

I could do `(cb: any) => { return; }` instead?

Reply:
What I sometimes do is `(cb: any) => { /*do nothing */ }`, and tslint is happy. Your solution works too!

Reply:
Ok, I'll stick to `/*do nothing */` for consistency, then :)